### ISSUE TITLE
Cannot find namespace 'fhirclient'

### ISSUE THREAD

**Issue by deepak-quo** [smart-on-fhir/client-js#63]
I import **'fhirclient'** node module in my angular project, it shows me this error message when I build the project. Please find the attachment for the error message. 
`import { oauth2 as SMART } from "fhirclient";`

<img width="906" alt="Screenshot 2019-09-16 at 5 13 18 PM" src="https://user-images.githubusercontent.com/54677731/64955324-96a35200-d8a5-11e9-8164-6c5d9e513ecc.png">

Please help with this issue.

**Comment by michaelgriffithus** [comment]
I also was having this exact same error with my ReactJS app.  Its something in your tsconfig.json file. I removed the contents of the file (not the file itself) and let the tsc regenerate the file with default values.  It fixed my problem.

**Comment by Nefari0uss** [comment]
@michaelgriffithus Do you mean delete the contents of the typings file? For reference, here is my `tsconfig`:

<details>
<summary>TS Config</summary>
```
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "strict": true,
    "jsx": "preserve",
    "importHelpers": true,
    "moduleResolution": "node",
    "experimentalDecorators": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "sourceMap": true,
    "baseUrl": ".",
    "types": [
      "webpack-env",
      "jest"
    ],
    "paths": {
      "@/*": [
        "src/*"
      ]
    },
    "lib": [
      "esnext",
      "dom",
      "dom.iterable",
      "scripthost"
    ]
  },
  "include": [
    "src/**/*.ts",
    "src/**/*.tsx",
    "src/**/*.vue",
    "tests/**/*.ts",
    "tests/**/*.tsx"
  ],
  "exclude": [
    "node_modules"
  ]
}
```
</details>

**Comment by vlad-ignatov** [comment]
This should be fixed in the latest version `2.1.1`. Apologies for the inconvenience and feel free to reopen if you are still having issues.

**Comment by Sanjeev-92** [comment]
i am implementing 
 oauth2ReadyCallback = (smartClient: fhirclient.SMART.SMARTClient) => {
    if (!this._smartClient) {
      this._smartClient = smartClient;
      this._smartClientSubject.next(smartClient);
    }
  }
Here   fhirclient.**SMART**.SMARTClient is not recognised please help me to solve this issue
